tee-worker/cli/lit_parentchain_nonce.sh

```diff
@@ -67,9 +61,9 @@ echo "* Send wrong parentchain extrinsic"
 ${CLIENT} trusted --mrenclave $MRENCLAVE --direct send-erroneous-parentchain-call
 echo ""
 
-sleep 10
+sleep 20
 # wait for 20 `ProcessedParentchainBlock` events, which should take around 2 min (2 workers)
 # if the incoming parentchain extrinsic is blocked (due to the wrong nonce), there won't be
 # such many events.
 set -e
-timeout -v --foreground 150s $CLIENT listen -e 20
\ No newline at end of file
+timeout -v --foreground 250s $CLIENT listen -e 10
```

Comment:
Any reason for this?

Reply:
I updated this to 
```
timeout -v --foreground 150s $CLIENT listen -e 10
``` 
Because there is a single worker based on the comments, 
```
# wait for 20 `ProcessedParentchainBlock` events, which should take around 2 min (2 workers)
# if the incoming parentchain extrinsic is blocked (due to the wrong nonce), there won't be
# such many events.
```